client/src/components/Submission.js

```diff
@@ -1,22 +1,37 @@
 import React, { Component } from 'react';
+import scriptLoader from 'react-async-script-loader'
+
+class Submission extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loaded: true
+    }
+  }
+
+  componentWillUnmount() {
+    window.location.reload();
+  }
 
-export default class Submission extends Component {
   render() {
 
+
     return (
       <div>
-        <h1>Hello from Submit Challenge Page!!!</h1>
+        <h1>10Hello from Submit Challenge Page!!! Google Client</h1>
 
-        <span id="signinButton" class="pre-sign-in">
+        <span id="signinButton" className="pre-sign-in">
           {/*<!-- IMPORTANT: Replace the value of the <code>data-clientid</code>
                attribute in the following tag with your project's client ID. -->*/}
+          {this.state.loaded ?
```

Comment:
Related to comment above -- if loaded is always set to true, then is this conditional needed?